pkgs/tools/networking/openfortivpn/default.nix

```diff
@@ -1,9 +1,12 @@
 { stdenv, lib, fetchFromGitHub, autoreconfHook, pkg-config
-, openssl, ppp
-, systemd ? null }:
+, openssl
+, ppp
+, systemd
+}:
 
 let
-  withSystemd = stdenv.isLinux && !(systemd == null);
+  withSystemd = stdenv.isLinux;
+  withPpp     = stdenv.isLinux;
```

Comment:
Normally we expose those options as inputs for convince.

Reply:
how can I make a such thing? can you show me an example?